### DISCUSSION TITLE
New Game Mode: Wrap-Around

### DISCUSSION THREAD

**Discussion by Maximtap** [BattlesnakeOfficial/feedback#24] [General]
Snakes can go through a wall now. When they do so, their head appears on the other side of the map. Might be too confusing to watch,

**Comment by Nettogrof** [comment]
Like warping around ?   Meaning that a snake can go always in 'right' direction, then appear to the left side of the board ?

**Reply by Maximtap** [reply]
Yeahhh that's right :D

**Comment by andrewdmaclean** [comment]
@Maximtap love this idea!

**Comment by bvanvugt** [comment]
This will likely be the next game mode release.

**Reply by Maximtap** [reply]
nice 👍

**Reply by bvanvugt** [reply]
Issue here for tracking release:
https://github.com/BattlesnakeOfficial/feedback/issues/66

**Comment by bvanvugt** [comment]
This is live now! Thanks @Maximtap for the suggestion. 

New 'wrapped' mode games can be created in the UI, and wrapped will be the official competitive mode throughout 2022 :)

More details:
* [Spring League Announcement](https://blog.battlesnake.com/spring-league-2022/)
* [Wrapped Mode Docs](https://docs.battlesnake.com/references/game-modes#wrapped)

**Reply by Maximtap** [reply]
Nice looks really cool
